chore: minor fixes, false positives ratio printed

M4tteoP opened the pull request.

fzipi commented:
Minor ask: can you add the time in seconds that it took to run in the json output?

We have this `totalTime`, which makes sense but it doesn't show properly:

```json
{"count":10000,"falsePositives":408,"falsePositivesPerRule":{"920220":198,"920221":198,"932235":4,"932270":2,"932380":2,"933160":1,"942100":1,"942230":1,"942360":1},"totalTime":4356441750}
```

M4tteoP commented:
PTAL, moved to Seconds and renamed the variable:
```
▶ go run main.go quantitative --crs-path ~/Repo/coreruleset --corpus-size "10k" -o json | jq .
11:45PM INF ⏳Running quantitative tests with 10 goroutines
{
  "count": 10000,
  "falsePositives": 10407,
  "falsePositivesPerRule": {
    "920220": 198,
    "920221": 198,
    "932235": 4,
    "932270": 2,
    "932380": 2,
    "933160": 10000,
    "942100": 1,
    "942230": 1,
    "942360": 1
  },
  "totalTimeSeconds": 1.63549225
}
```

fzipi reviewed (approved):
LGTM
